wiki/Tournaments/Official_support/en.md

```diff
@@ -90,6 +91,17 @@ Once the tournament has concluded, the tournament organisers will need to submit
 - An appropriately designed badge ([see below](#profile-badges)).
 - A link to the publicly viewable catalogue of your tournament's match history.
 
+#### Referee bots/automation
+
+The use of bots/automation to aid in the running of officially supported tournament matches is permitted *with some caveats to reflect their cutting-edge nature*, so long as any applicable software/workflows meets the requirements listed below:
+
+- The bot/automation must be made publicly and functionally available as an open-source repository.
+- The bot/automation must be attached to and run from a singular [Bot account](/wiki/Bot_account), following all stipulations and requirements listed therein.
```

Comment:
The requirements for a traditional bot account in that case are too strict. There's nothing stopping lobbies to be ran from user accounts so that should be allowed, however recommendations from the Bot account article should be followed.